TenureInformationApi.Tests/V1/Controllers/TenureInformationControllerTests.cs

```diff
@@ -30,18 +31,31 @@ public TenureInformationControllerTests()
         {
             _mockGetByIdUsecase = new Mock<IGetByIdUseCase>();
             _mockPostTenureUseCase = new Mock<IPostNewTenureUseCase>();
+            _mockUpdateTenureUseCase = new Mock<IUpdateTenureUseCase>();
             _mockTokenFactory = new Mock<ITokenFactory>();
             _mockContextWrapper = new Mock<IHttpContextWrapper>();
             _mockHttpRequest = new Mock<HttpRequest>();
-            _classUnderTest = new TenureInformationController(_mockGetByIdUsecase.Object, _mockPostTenureUseCase.Object, _mockTokenFactory.Object,
+            _classUnderTest = new TenureInformationController(_mockGetByIdUsecase.Object, _mockPostTenureUseCase.Object, _mockUpdateTenureUseCase.Object, _mockTokenFactory.Object,
                 _mockContextWrapper.Object);
             _mockContextWrapper.Setup(x => x.GetContextRequestHeaders(It.IsAny<HttpContext>())).Returns(new HeaderDictionary());
 
         }
 
-        private GetByIdRequest ConstructRequest(Guid? id = null)
+        private TenureQueryRequest ConstructRequest(Guid? id = null)
         {
-            return new GetByIdRequest() { Id = id ?? Guid.NewGuid() };
+            return new TenureQueryRequest() { Id = id ?? Guid.NewGuid() };
+        }
+
+        private TenureQueryRequest ConstructQuery(Guid? id = null, Guid? personId = null)
+        {
+            return new TenureQueryRequest() { Id = id ?? Guid.NewGuid(), PersonId = personId ?? Guid.NewGuid() };
```

Comment:
Is there a reason you want this to only happen in tests? If not, you should also do this in the constructor.